src/Type/ObjectType/RootQuery.php

```diff
@@ -197,10 +197,21 @@ public static function register_type() {
 										'check_enabled'  => false,
 									]
 								);
-								$post_id   = ! empty( $revisions ) ? array_values( $revisions )[0] : null;
+								$post_id   = ! empty( $revisions ) ? array_values( $revisions )[0] : $post_id;
 							}
 
-							return ! empty( $post_id ) ? $context->get_loader( 'post' )->load_deferred( $post_id ) : null;
+							$allowed_post_types   = get_post_types( [ 'show_in_graphql' => true ] );
```

Comment:
@jasonbahl I wasn't sure whether the next several lines here were actually part of this bug fix or if they were part of your other feature branch. They seem to make sense in this context so I left them in place, but we could probably revert back to the one-liner return statement in this PR if the `allowed_post_types` check is coming with your `contentTypeName` feature.